publish-service/src/test/java/com/ericsson/eiffel/remrem/publish/service/MessageServiceRMQImplUnitTest.java

```diff
@@ -65,13 +65,15 @@ public class MessageServiceRMQImplUnitTest {
     private static final String exchangeName= "amq.direct";
     private static final String domainId= "eiffelxxx";
     private boolean createExchangeIfNotExisting = true;
+    private static final Integer tcpTimeOut = 5000;
 
     @PostConstruct public void setUp() throws Exception {
         rmqHelper.getRabbitMqPropertiesMap().put(protocol, new RabbitMqProperties());
         rmqHelper.getRabbitMqPropertiesMap().get(protocol).setProtocol(protocol);
```

Comment:
Is it necessary to call rmqHelper.getRabbitMqPropertiesMap().get(protocol) over and over? I guess the result is the RabbitMqProperties instantiated at line 71.

Reply:
There is another way like, assigning the protocol value to the some other variable..
RabbitMqProperties rabbitMqProp = rmqHelper.getRabbitMqPropertiesMap().get(protocol);
rabbitMqProp.setHost(host);

Reply:
> There is another way like, assigning the protocol value to the some other variable.. RabbitMqProperties rabbitMqProp = rmqHelper.getRabbitMqPropertiesMap().get(protocol); rabbitMqProp.setHost(host);

Yes, but why first use ...put(protocol, new new RabbitMqProperties()) and then ...get(protocol)?
I'd 
   RabbitMqProperties properties = new RabbitMqProperties());
   rmqHelper.getRabbitMqPropertiesMap().put(protocol, properties);
   properties.setProtocol(protocol);
   properties.setHost(host);
   ...